.changeset/clean-needles-deliver.md

```diff
@@ -0,0 +1,5 @@
+---
+"stylelint": patch
+---
+
+Fixed: false positive CJS deprecation warning for dual plugin packages
```

Comment:
nit

there's a word missing after "dual"

Reply:
Oh sorry, I'm not a native English speaker, can you point out what word should I add or how should I change the sentence?

Reply:
I am not really sure, maybe `…for hybrid plugin packages` would be better?
Or perhaps `…for dual-package plugins`?
Either one sounds better to me.

Reply:
```suggestion
Fixed: false positive CJS deprecation warning for dual-package plugins
```